UPGRADING

```diff
@@ -323,7 +323,8 @@ PHP 8.4 UPGRADE NOTES
     Previously, the return type was bool but only true could be returned in practice.
 
 - Gettext:
-  . bind_textdomain_codeset now throws an exception if the domain's argument is empty.
+  . bind_textdomain_codeset, textdomain and d(*)gettext functions now throw an exception
+    if the domain's argument is empty.
```

Comment:
Language nit/question:
"if the domain's argument is empty", I guess it's better to drop the `'s` ? It's not like the domain "owns" an argument. But I'm not sure.

Reply:
fair point